skillmap/src/components/AppModal.tsx

```diff
@@ -347,17 +350,54 @@ export class AppModalImpl extends React.Component<AppModalProps, AppModalState>
                         </div>
                     </div>
                 })}
-                <div className="sign-in-remember" onClick={() => {
+                <div className="sign-in-remember checkbox" onClick={() => {
                     const rememberMe = !rememberMeSelected;
                     tickEvent("skillmap.signindialog.rememberme", { rememberMe: rememberMe.toString() });
-                    this.setState({ rememberMe });
+                    this.setState({ checkboxSelected: rememberMe });
                 }}>
                     <i className={`icon square outline ${rememberMeSelected ? "check" : ""}`} />
                     {lf("Remember me")}
                 </div>
             </div>
         </Modal>
     }
+
+    renderDeleteAccountModal() {
+        const { checkboxSelected } = this.state;
+
+        const buttons = [];
+        buttons.push({
+            label: lf("Confirm"),
+            className: checkboxSelected ? "confirm" : "confirm disabled",
+
+            onClick: async () => {
+                if (checkboxSelected) {
+                    tickEvent("skillmap.profile.delete", { link: "https://aka.ms/cloudsave" });
```

Comment:
why do we need this link in the tick event?

Reply:
I think it was just copied from the tick in webapp. Probably don't need it in either place. Not useful data.

Reply:
! i will remove in both